pt/endpoints/address/resource.py

```diff
@@ -0,0 +1,57 @@
+from datetime import date
+from flask import jsonify
+from flask import g as g_ami
+from flask_restful import Resource
+from flask_httpauth import HTTPTokenAuth
+from utils.logging import logging
+from utils.oauth import auth, g
+from .model import address, AMI
+
+# pylint: disable=C0103
+auth_ami = HTTPTokenAuth(scheme='Bearer')
+# pylint: enable=C0103
+@auth_ami.verify_token
+def verify_token(token):
+    # get username and uuid from database
+    ami = address(token, date.today())
+    if ami:
+        g_ami.uuid = ami.uuid
+        g_ami.name = ami.name
+        g_ami.address = ami.iota_address
+        return True
+    return False
+
+
+class AddressResource(Resource):
+    # pylint: disable=R0201
+    @auth_ami.login_required
+    def get(self):
+        logging.info(
+            "[Get Address Request]\nUser name:%s\nUUID:%s\nIOTA Address:%s"
+            % (g_ami.name, g_ami.uuid, g_ami.address)
+        )
```

Comment:
You can simply use 
```
f"[Get Address Request]\nUser name:{g_ami.name}\nUUID:{g_ami.uuid}\nIOTA Address:{g_ami.address}"
```

in python3.

Reply:
After running on a new server with Python≥3.5, all print codes will use this method.
Thanks for suggestion! : )